addon/templates/components/frost-guage.hbs

```diff
@@ -0,0 +1,36 @@
+<svg height='{{size}}' width='{{size}}' viewPort='0 0 {{size}} {{size}}' version='1.1' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
If the element is an svg do we want to make the component an svg instead of a div? We will be able to bind things like height, width and size.